tests/Test/AbstractFixerTestCase.php

```diff
@@ -110,11 +111,12 @@ protected function getTestFile($filename = __FILE__)
      * This method throws an exception if $expected and $input are equal to prevent test cases that accidentally do
      * not test anything.
      *
-     * @param string            $expected The expected fixer output
-     * @param null|string       $input    The fixer input, or null if it should intentionally be equal to the output
-     * @param null|\SplFileInfo $file     The file to fix, or null if unneeded
+     * @param string                      $expected    The expected fixer output
+     * @param null|string                 $input       The fixer input, or null if it should intentionally be equal to the output
+     * @param null|\SplFileInfo           $file        The file to fix, or null if unneeded
+     * @param null|WhitespacesFixerConfig $whitespaces The white space configuration, or null to use the default
      */
-    protected function doTest($expected, $input = null, \SplFileInfo $file = null)
+    protected function doTest($expected, $input = null, \SplFileInfo $file = null, WhitespacesFixerConfig $whitespaces = null)
```

Comment:
No need to add this argument IMO. The tested fixer is available via the `fixer` property of the class, so a test can call `$this->fixer->setWhitespacesConfig(...)` before `$this->doTest(...)`.